_component_design/button-drop-down.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Drop-Down Buttons
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: >
+  A drop-down button is an element that reveals additional content when clicked by a user. There are two types of drop-down buttons: a basic drop-down button and a split drop-down button.
+preview-image: preview-images/buttons.svg
+status: stable
+resource: true
+last-modified: 2018-06-12
+helix-ui-css: true
+helix-ui-javascript: false
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use a basic drop-down button when there are multiple actions associated with a button but there is no designated primary action. When the user clicks a drop-down button, all actions appear in an actions menu.
+
+Use a split drop-down button when there are multiple actions associated with a button but only one primary action. The primary action is visible and available on the button, while the secondary actions are accessed through the drop-down menu. See the [actions menu]({{site.baseurl}}/components/actions-menu.html) for drop-down menu specifications.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+In addition to the best practices outlined in [simple buttons]({{site.baseURL}}/components/buttons.html) and [actions menu]({{site.baseURL}}/components/actions-menu.html), consider the following best practices when you construct a drop-down button:
+
+- While a drop-down button reduces the clutter on a page, drop-down buttons require the user to complete more steps. Consider listing all options as an alternative to concealing options within a button.
+- xxx.
```

Comment:
could use wordsmithing: 
Be sure that the basic drop-down button is labelled with an informative label. 

not sure what i mean by informative, but a user should be able to predict what will be in the dropdown.